static/js/VersionBlock.jsx

```diff
@@ -171,6 +171,9 @@ class VersionBlock extends Component {
     }
   }
   makeVersionNotes(){
+    if (!this.props.showNotes) {
+      return null;
+    }
```

Comment:
Do we also want to have `vnotes && <span className="" dangerouslySetInnerHTML={ {__html: vnotes} } />`
as to not render that element at all in line 301?